tests/acceptance/password-test.js

```diff
@@ -53,7 +59,9 @@ test('visiting /password/forgot', function(assert) {
     assert.equal(currentURL(), '/password/forgot');
   });
 
-  passwordPage.forgotPasswordForm.sendForgotPasswordSuccessfully('admin@gmail.com');
+  andThen(() => {
```

Comment:
It's because the Mirage response is a promise and we don't know when it will return.